### ISSUE TITLE
Deep Neural Network Models, matrix factorization and embedding

### ISSUE THREAD

**Issue by Shicheng-Guo** [Shicheng-Guo/rbiotools#40]
https://developers.google.com/machine-learning/recommendation/dnn/softmax
